examples/favorites_advanced/getx_favorites_advanced/lib/feature_home/views/home_page.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:badges/badges.dart';
+import 'package:curved_navigation_bar/curved_navigation_bar.dart';
+import 'package:getx_favorites_advanced/base/ui_components/puppies_app_bar.dart';
+import 'package:getx_favorites_advanced/feature_home/controllers/navbar_controller.dart';
+import 'package:getx_favorites_advanced/feature_puppy/favorites/controllers/favorite_puppies_controller.dart';
+import 'package:getx_favorites_advanced/feature_puppy/favorites/views/favorites_page.dart';
+import 'package:getx_favorites_advanced/feature_puppy/search/search_page.dart';
+import 'package:favorites_advanced_base/models.dart';
+import 'package:favorites_advanced_base/extensions.dart';
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final NavbarController navController = Get.put(NavbarController());
+    final FavoritePuppiesController favController =
```

Comment:
@RomanovaPrime, please make sure the static analysis is enabled on your IDE, the var *favController* is not used.

Reply:
Removed unnecessary variable.